app/web_framework/cluster_slave_application.py

```diff
@@ -116,6 +126,82 @@ def get(self, build_id, subjob_id):
         self.write(response)
 
 
+class _AtomsHandler(_ClusterSlaveBaseAPIHandler):
+    def get(self, build_id, subjob_id):
+        response = {
+            'comment': 'not implemented',
+            }
+        self.write(response)
+
+
+class _AtomHandler(_ClusterSlaveBaseAPIHandler):
+    def get(self, build_id, subjob_id, atom_id):
+        response = {
+            'comment': 'not implemented',
+            }
+        self.write(response)
```

Comment:
Could you just do a `pass` similar to `_SubjobsHandler` above? I'm not sure what the reasoning behind the original "not implemented" response was.